lib/kafka_ex/server_0_p_8_p_2.ex

```diff
@@ -186,9 +188,15 @@ defmodule KafkaEx.Server0P8P2 do
     offset_commit_request = %{offset_commit_request | consumer_group: consumer_group}
 
     offset_commit_request_payload = OffsetCommit.create_request(state.correlation_id, @client_id, offset_commit_request)
+
     response = broker
-      |> NetworkClient.send_sync_request(offset_commit_request_payload, config_sync_timeout())
-      |> OffsetCommit.parse_response
+      |> NetworkClient.send_sync_request(
+           offset_commit_request_payload,
+           config_sync_timeout())
+      |> case do
+           {:error, reason} -> {:error, reason}
+           response -> OffsetFetch.parse_response(response)
```

Comment:
I think this should be OffsetCommit?

Reply:
Yup. My Bad. I have updated the PR. Thanks.